test/new-e2e/tests/windows/common/service.go

```diff
@@ -177,3 +178,14 @@ func GetServiceAccountName(host *components.RemoteHost, service string) (string,
 	out, err := host.Execute(cmd)
 	return strings.TrimSpace(out), err
 }
+
+// GetServicePID returns the PID of the service
+func GetServicePID(host *components.RemoteHost, service string) (int, error) {
+	cmd := fmt.Sprintf("(Get-WmiObject Win32_Service -Filter \"Name=`'%s`'\").ProcessId", service)
+	out, err := host.Execute(cmd)
+	if err != nil {
+		return 0, err
+	}
+	out = strings.TrimSpace(out)
+	return strconv.Atoi(out)
+}
```

Comment:
this is great, we'll be able to use this on (eventual) USM and Service Discoverability tests👏